app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,39 @@
 <html>
   <head>
     <title>BillingPaymentSystem</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name='viewport' content='width=device-width,initial-scale=1'>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
-  </body>
+     <% if user_signed_in? && current_user.role == 'admin' %>
+       <%= render partial: 'shared/navbar_admin' %>
+     <% elsif user_signed_in? %>
+       <%= render partial: 'shared/navbar_buyer' %>  
+     <% end %>
+     <div class='container'>
```

Comment:
move following logic to a partial

application_helper.rb
def nav_bar
return 'shared/navbar_admin' if current_user.admin?

'shared/navbar_buyer'
end

application.html.erb
<%= render nav_bar unless  user_signed_in? %>